<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDateDisplayGeneralDlg</class>
 <widget class="QWidget" name="QDateDisplayGeneralDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文 本 色：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>排列</string>
   </property>
   <widget class="QRadioButton" name="Alignment_Left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>居左</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Alignment_Center">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>居中</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Alignment_Right">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>居右</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>字    体：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Format">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YY MM DD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>YYYY MM DD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MM DD YY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MM DD YYYY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DD MM YY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DD MM YYYY</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>格    式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>字体大小：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Pattern">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Frame_Color">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>40</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="FG_Color">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>71</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>71</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>前景色：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BG_Color">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>101</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>样  式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>边框色：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>背景色：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadShape">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>外形</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="LView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Text_Color">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>分 隔 符：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Separative_Sign">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="FontSize">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Font">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ID</tabstop>
  <tabstop>LView</tabstop>
  <tabstop>Pattern</tabstop>
  <tabstop>Format</tabstop>
  <tabstop>Separative_Sign</tabstop>
  <tabstop>Font</tabstop>
  <tabstop>FontSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
